app.config.js

```diff
@@ -141,7 +141,6 @@ module.exports = function (config) {
         adaptiveIcon: {
           foregroundImage: './assets/icon-android-foreground.png',
           monochromeImage: './assets/icon-android-foreground.png',
-          backgroundImage: './assets/icon-android-background.png',
```

Comment:
This is the one thing I don't like. Going to ask in Expo channel what might have changed. Fails to build if I leave this here.